cluster/gce/util.sh

```diff
@@ -1354,3 +1356,8 @@ function ssh-to-node {
 function prepare-e2e() {
   detect-project
 }
+
+# Writes configure-vm.sh to a temporary location with comments stripped.
```

Comment:
You should expand this comment a bit to explain why we are doing this (because otherwise we are going to overflow the size of a single metadata field). This is the simplest minification that we can do and reduces the number of bytes by about a third.